### DISCUSSION TITLE
Basics?

### DISCUSSION THREAD

**Discussion by macdroid53** [j8sr0230/fc_nodes#122] [Q&A]
Sorry, I'm the type that needs to start with simple examples.

How to do:

Create a face.

Get the normal of a face.

Given a Box, how to get a single face.

I think a lot of the outputs are Python lists. How to index through the list.

Documentation of the node inputs and outputs? Some are obvious; others, not so much.

**Comment by DaniloMitrovic** [comment]
So far this seems to be modeled by "grasshopper" or other visual programming languages that ship with CAD/Music software. Till proper tutorials are done, perhaps this could help [10+ h of grasshopper basics.](https://www.youtube.com/watch?v=b0elmzjWlE8)
